Library/Homebrew/extend/blank.rb

```diff
@@ -13,15 +12,13 @@ class Object
   # to
   #
   #   address.blank?
-  #
-  # @return [true, false]
+  sig { returns(T::Boolean) }
   def blank?
-    respond_to?(:empty?) ? !!empty? : !self
+    respond_to?(:empty?) ? !!T.unsafe(self).empty? : false
```

Comment:
Is the `T.unsafe` only necessary here because we're using `typed: strict`? If so, maybe it'd be better to just use `typed: true`.

Reply:
Removing the `T.unsafe` will cause a type error even at `typed: true`, as `respond_to?` is not a supported type guard (it can't check the arity of the parameters, let alone their types).